spec/requests/v6/assessments_controller_spec.rb

```diff
@@ -1100,7 +1100,7 @@ def cash_transactions(amount)
 
             it "has outgoings_housing_cost" do
               expect(remarks.fetch(:outgoings_housing_cost)).to eq(
-                { unknown_frequency: ["hc-r-1"] },
+                { unknown_frequency: ["hc-r-1", "hc-r-1"] },
```

Comment:
@starswan We have duplicate elements in an array. Is it a type?

Reply:
No - its some proof that the changes worked and we are getting remarks for partner